go/libkb/pseudonym.go

```diff
@@ -260,3 +258,12 @@ func checkTlfPseudonymFromServer(ctx context.Context, g *GlobalContext, req TlfP
 
 	return nil
 }
+
+func RandomHmacKey() [32]byte {
+	key := [32]byte{}
+	n, err := cryptorand.Read(key[:])
```

Comment:
I could have sworn we already had a helper function that did both checks, but it might just be in KBFS.